docker/Dockerfile.base

```diff
@@ -5,7 +5,7 @@
 # to build and run the Docker image.
 # This image is based on the container-registry.oracle.com/os/oraclelinux:9-slim image and contains the following
 # components:
-#   Python3.11.2 (with pip 22.3.1 and setuptools 65.5.1) compiled and installed from source.
+#   Python3.11.4 (with pip 22.3.1 and setuptools 65.5.1) compiled and installed from source.
```

Comment:
Please upgrade `pip` and `setuptools` to the latest versions too.

Reply:
Actually, because we have decided back then that we don't want to install pip and setuptools separately. We have opted to install pip together with the Python installation. I will remove this note on pip and setuptools.
After having a look [here](https://github.com/oracle/macaron/blob/d6f1cc60a5eb1c26165f530afc73ea735e800e38/docker/Dockerfile.base#L24C5-L28C10) I have noticed that these env variables are not used anywhere (they are used back when we still install pip and setuptools separately from the Python installation - following the method [here](https://github.com/docker-library/python/blob/f568f56f28fab0fe87b34db777e2c2861cef002b/3.11/slim-buster/Dockerfile#L133)).
Therefore, I have also removed them and make sure that pip and setuptools are upgraded to the latest version in `Dockerfile.final` - see https://github.com/oracle/macaron/pull/379/commits/3f68c4f73bf717874e45fff416370a0dd84c0053) and https://github.com/oracle/macaron/pull/379/commits/3fffadeb341f13e82fb8325d52a67bd2abb6d2db

Reply:
I was actually confused because we had indeed decided to install the latest version of `pip` and `setuptools` for the corresponding Python version. Thanks for the clarification.